packages/app/components/Settings/ChatbotSettings.tsx

```diff
@@ -294,13 +267,20 @@ export default function ChatbotSettings(): ReactElement {
               <>
                 <Form.Item
                   name="files"
-                  rules={[
-                    {
-                      required: true,
-                      message: 'Please provide document files.',
-                    },
-                  ]}
+
+                  // rules={[
```

Comment:
If not used, can we remove this?

Why not add a rule to support specific file extensions?

Reply:
the rules do exist for the dragger, however we do not have control over that for github url and what types of files that url upload link contains